pkg/parse/source.go

```diff
@@ -95,10 +95,33 @@ func (o *files) readConfigFiles(state *sourceState) status.Error {
 	if err != nil {
 		return status.PathWrapError(errors.Wrap(err, "listing files in the configs directory"), syncDir.OSPath())
 	}
+
+	commitChanged, newCommit, err := o.checkCommitChanged(p, state)
+	if err != nil {
+		return status.TransientError(err)
```

Comment:
Is this necessarily a transient error? This scenario seems like an unexpected error which could potentially not be transient

Reply:
No longer calling hydrate.sourceCommitAndDir in checkCommitChanged, so error should be transient.